src/index.js

```diff
@@ -175,12 +177,22 @@ function start() {
       close(); // close the app for Linux and Windows
     }
   });
+
+  if (OS === 'win32' || OS === 'darwin') {
+    // Check for new updates
+    const autoUpdater = new AutoUpdater({
```

Comment:
I'm guessing that this needs to run later than what we used to do (at the very beginning) because we rely on the render process for user confirmation

Reply:
that's correct